src/app/pages/home/components/home-action-button.tsx

```diff
@@ -3,16 +3,16 @@ import { memo } from 'react';
 import { Box, ButtonProps, Text } from '@stacks/ui';
 
 interface HomeActionButtonProps extends ButtonProps {
-  icon: any;
+  icon?: any;
   label: string;
   buttonComponent(props: ButtonProps): React.JSX.Element;
 }
 export const HomeActionButton = memo((props: HomeActionButtonProps) => {
   const { icon, label, buttonComponent: Button, ...rest } = props;
 
   return (
-    <Button height="36px" px="base-tight" py="tight" position="relative" {...rest}>
-      <Box as={icon} mr="tight" size="14px" />
+    <Button height="36px" minWidth="80px" px="base-tight" py="tight" position="relative" {...rest}>
```

Comment:
`{icon &&`?

Reply:
Why better?

Reply:
@kyranjamie haven't you asked me not to use `&&` like this ...getting confused by all the CR pattern preferences.

Reply:
Maybe it is a pattern I am just not used to. I like `&&` as you don't need a `:null` then. It's hardly a big deal though

Reply:
Haha, yeah, no worries. I used to use it a lot and preferred it too, but felt I was asked to move away from it so wanted to clarify.

Reply:
Got confused here because the discussion is on line 14 in relation to code the line below.

Only con for `&&` is that in some cases, say you're using a number, `0` is falsy so can have unintended consequences, but that can happen with ternary if you get expression wrong as well. 

Honestly, don't really mind here. Maybe `??` would actually be the best operator in most cases.

Reply:
@pete-watters I have noticed this with a lot of your comments, they don't seem to show up on the right line?

Reply:
I'll take more care to get them on the right place in future 👍